include/revng/AutoEnforcer/AutoEnforcerLibraryRegistry.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
```

Comment:
Missing header comment.
Please review all files. Maybe you can add a rule to `check-conventions`.